### PR TITLE
Fix Namespace `exists` method typing

### PR DESCRIPTION
**Pull request by Scrumplex**
See example response on https://docs.gitlab.com/ee/api/namespaces.html#get-existence-of-a-namespace

### CODE REVIEW

File: packages/core/src/resources/Namespaces.ts [hunk 1]
```diff
@@ -37,12 +42,12 @@ export class Namespaces<C extends boolean = false> extends BaseResource<C> {
   }
 
   exists<E extends boolean = false>(
-    namespaceId: string | number,
+    namespace: string | number,
```
Comment (jdalrymple) [inline packages/core/src/resources/Namespaces.ts:45]: If you can remove the number type here, I can merge this ASAP 🙏

**Comment by jdalrymple** [comment]
Thanks for the catch!

File: packages/core/src/resources/Namespaces.ts (hunk 1 above)
Comment (Scrumplex) [reply packages/core/src/resources/Namespaces.ts]: done! :D

**Comment by jdalrymple** [comment]
:rocket: PR was released in [`40.4.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/40.4.0) :rocket:

**Pull request merged**